website/app/styles/pages/application/cover.scss

```diff
@@ -63,7 +63,7 @@
   height: min-content;
   margin-top: 24px;
   overflow-x: auto;
-  overflow-y: visible; // needed or it will show a small vertical scrollbar
+  overflow-y: hidden; // .doc-page-tabs__tab--is-current has an ::after pseudoelement that always overflows with 3px, so we disable the vertical scrollbar and rely on the horizontal one
```

Comment:
I am not sure why, when I implemented it, using `hidden` didn't work (you see it from my explicit comment added to the line), 🤷‍♂️